Resources/Prototypes/_Goobstation/Entities/Clothing/Eyes/glasses.yml

```
@@ -16,3 +16,62 @@
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesBroken
+  name: broken glasses
+  description: My glasses! I can't see without my glasses!
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Clothing/Eyes/Glasses/brokenglasses.rsi
+  - type: Clothing
+    sprite: _Goobstation/Clothing/Eyes/Glasses/brokenglasses.rsi
+  - type: Tag
+    tags:
+      - Trash
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)